core/src/models/Import.ts

```diff
@@ -32,22 +32,13 @@ export interface ImportRecordProperties {
 
 const IMPORT_CREATORS = ["run"] as const;
 
-const STATES = [
-  "associating",
-  "importing",
-  "processing",
-  "failed",
-  "complete",
-] as const;
+const STATES = ["associating", "importing", "failed", "complete"] as const;
 
 const STATE_TRANSITIONS: StateMachine.StateTransition[] = [
   { from: "associating", to: "failed", checks: [] },
   { from: "associating", to: "importing", checks: [] },
   { from: "importing", to: "failed", checks: [] },
   { from: "importing", to: "complete", checks: [] },
-  { from: "importing", to: "processing", checks: [] },
-  { from: "processing", to: "failed", checks: [] },
-  { from: "processing", to: "complete", checks: [] },
```

Comment:
There is no more `processing` state for imports.  They are `complete` once the record they are associated to goes ready.